samples/k8s-offer-azure-vote-custom-meters/azure-vote-custom/values.yaml

```diff
@@ -17,7 +17,7 @@ global:
       resourceId: "DONOTMODIFY" # application's Azure Resource ID, Azure populates this value at deployment time
     images:
       frontend:
-        digest: sha256:fed588cc86c00250bd25219e727f31927f377f7b3863ae130e98e936bea92d02
+        digest: sha256:c6e5a59d7789155b291f473c48520a8c83438488d3531e3fbf8f854f0be79d99
```

Comment:
Should we also update the backend image to sha256:e600a259cb3ef2c6ceace477022d6159c3e72763c694d54c0263365101de3266 ? for azure vote app as well ?